### DISCUSSION TITLE
Why composer  doesn't have version command like npm :thinking:

### DISCUSSION THREAD

**Discussion by AshrafBasry** [composer/composer#12170] [General]
I always wondered why composer doesn't have version command like `npm`. I think it will be easier for package development , you make changes and then
```bash
composer version patch
composer version minor
composer version major
```
And it will be easier to identify a package's version just by looking into `composer.josn` version entry

what is the current approach for versioning composer packages?

**Comment by Seldaek** [comment]
Composer versioning is purely based on git tags so this does not really fit in Composer like it does in npm..
